src/eth/mining.json

```
@@ -50,32 +50,32 @@
 	},
 	{
 		"name": "eth_submitWork",
-		"summary": "Used for submitting a proof-of-work solution.",
+		"summary": "Can be used by an external miner to submit their proof-of-work solution.",
```

```suggestion
		"summary": "Use for submitting a proof-of-work solution.",
```
I'm not sure if the update improve the clarity of the summary enough to warrant a change.

---

ok. I just got it from the comments above the function https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/consensus/ethash/api.go#L66-L84

---

Reverted